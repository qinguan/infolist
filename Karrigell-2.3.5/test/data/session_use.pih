<h3>Session test goes on</h3>
<%
session=Session()
print session.name
print session.firstname
session.close()
%>
