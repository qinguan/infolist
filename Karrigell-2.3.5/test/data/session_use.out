<h3>Session test goes on</h3>
Karrigell
Python
